aries-site/src/utils/themeMode/useEventListener.js

```diff
@@ -0,0 +1,21 @@
+import { useRef, useEffect } from 'react';
+
+export const useEventListener = (eventName, handler, element = global) => {
+  const savedHandler = useRef();
+
+  useEffect(() => {
+    savedHandler.current = handler;
+  }, [handler]);
+
+  useEffect(() => {
+    const isSupported = element && element.addEventListener;
+    if (!isSupported) return;
+
+    const eventListener = event => savedHandler.current(event);
+    element.addEventListener(eventName, eventListener);
+    // eslint-disable-next-line consistent-return
+    return () => {
+      element.removeEventListener(eventName, eventListener);
+    };
+  }, [eventName, element]);
+};
```

Comment:
This looks duplicative of `/hooks/useEventListener.js` (except for the `eslint-disable`) --- Remove one or the other.